source/opt/ir_loader.cpp

```diff
@@ -136,6 +140,23 @@ bool IrLoader::AddInstruction(const spv_parsed_instruction_t* inst) {
         }
         function_->AddParameter(std::move(spv_inst));
       } else {
+        if (opcode == SpvOpExtInst &&
+            spvExtInstIsDebugInfo(inst->ext_inst_type)) {
+          const uint32_t ext_inst_index = inst->words[4];
+          const OpenCLDebugInfo100Instructions ext_inst_key =
+              OpenCLDebugInfo100Instructions(ext_inst_index);
+          if (ext_inst_key != OpenCLDebugInfo100DebugScope &&
```

Comment:
What happens if this code is executed with a module using the older DebugInfo extended instruction set? 
Does it work by accident? fail by accident?
Please add a comment about what is expected to work.  And if it's easy to make work with DebugInfo, then please do so.